Gem/Code/Source/Components/PerfTest/NetworkTestSpawnerComponent.cpp

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <NetworkPrefabSpawnerInterface.h>
+#include <AzCore/Component/TransformBus.h>
+#include <AzCore/Math/Random.h>
+#include <Components/NetworkRandomComponent.h>
+#include <Components/PerfTest/NetworkTestSpawnerComponent.h>
+#include <LmbrCentral/Shape/ShapeComponentBus.h>
+#include <Multiplayer/IMultiplayer.h>
+#include <Multiplayer/Components/NetBindComponent.h>
+#include <Source/AutoGen/NetworkRandomComponent.AutoComponent.h>
+#include "NetworkPrefabSpawnerComponent.h"
+
+namespace MultiplayerSample
+{
+    NetworkTestSpawnerComponentController::NetworkTestSpawnerComponentController(NetworkTestSpawnerComponent& parent)
+        : NetworkTestSpawnerComponentControllerBase(parent)
+    {
+    }
+
+    void NetworkTestSpawnerComponentController::OnActivate([[maybe_unused]] Multiplayer::EntityIsMigrating entityIsMigrating)
+    {
+        AZ::TickBus::Handler::BusConnect();
```

Comment:
You could actually use an AZ::ScheduledEvent and avoid any hooking up to tickbus or managing accumulators and such.. you'd just have a single event that gets popped and invokes your callback every N milliseconds.